build.sbt

```diff
@@ -235,7 +235,7 @@ def updateWebsiteTag =
 
 lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
   organization := "io.getquill",
-  scalaVersion := "2.12.3",
+  scalaVersion := "2.11.11",
```

Comment:
Some 2.12 features are not supported in 2.11 so it's easy for dev to make a compilation mistake, which could be caught only in PR build. To prevent this, I'd suggest to use older scala version as default